scripts/setup_staking.sh

```diff
@@ -307,10 +308,33 @@ function copy_cliain_to_pod() {
   cliain_path="$1"
   validator_pod_name="$2"
   namespace="$3"
-  info "Copying binary to validator's pod ${validator_pod_name}:${CLIAIN_PATH_ON_POD}"
-	kubectl cp -n "${namespace}" "${cliain_path}" "${validator_pod_name}":"${CLIAIN_PATH_ON_POD}" || \
-	  error "Failed to copy cliain binary to ${validator_pod_name}:${CLIAIN_PATH_ON_POD}"
-	prompt_if_interactive_mode "Press enter to continue"
+
+  if  [ -z "${CLIAIN_NO_COPY}" ]; then
```

Comment:
This probably should be around the call of this function rather than inside of it, for consistency with all the other variables.

Reply:
Good point, done.